src/components/Modal/Modal.module.css

```diff
@@ -0,0 +1,72 @@
+.overlay {
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
```

Comment:
You may use shorthand ```inset``` property instead

Reply:
Thanks for showing this CSS property, shorthand handy!
But for readability I will prefer to leave it as is.